src/transformers/utils/fx.py

```diff
@@ -119,7 +140,11 @@ def _generate_supported_model_classes(
         _REGULAR_SUPPORTED_MODELS.extend(_generate_supported_model_classes(item))
 
 _SPECIAL_SUPPORTED_MODELS = [
```

Comment:
This should be a list of strings, as you are making this file relying on more and more models. It will then make it impossible to use once you start using models that require special dependencies.